src/Asset/BunManager.php

```
@@ -2,59 +2,45 @@
+        return 'bun';
     }
 
     public function isInstalled(): bool
     {
         return parent::isInstalled() && file_exists($this->getLockFilePath());
     }
 
-    protected function getVersionCommand(): string
+    protected function getInstallCommand(): string
     {
         $command = Platform::isWindows() ? 'bun.exe' : 'bun';
```

PHPMD (2.15.0)

Avoid using static access to class '\Composer\Util\Platform' in method 'getInstallCommand'. (undefined)

(StaticAccess)